src/ts/components/seekbar.ts

```diff
@@ -138,18 +166,39 @@ export class SeekBar extends Component<SeekBarConfig> {
     // (the call must be up here to be executed for the volume slider as well)
     this.setPosition(this.seekBarBackdrop, 100);
 
+    // Add seekbar controls to the seekbar
+    const seekBarController = new SeekBarController(this.config.keyStepIncrements, player, uimanager.getConfig().volumeController);
+
+    seekBarController.setSeekBarControls(this.getDomElement(), this.getSeekBarType);
+
     if (!configureSeek) {
+      this.seekBarType = SeekBarType.Volume;
+      this.setAriaSliderMinMax('0', '100');
+      this.getDomElement().attr('aria-label', i18n.performLocalization(i18n.getLocalizer('settings.ariaLabel.volume')));
+      this.getDomElement().attr('aria-valuenow', `${player.getVolume()}`);
+      this.getDomElement().attr('aria-valuetext', `${i18n.performLocalization(i18n.getLocalizer('seekBar.value'))}: ${player.getVolume()}`);
       // The configureSeek flag can be used by subclasses to disable configuration as seek bar. E.g. the volume
       // slider is reusing this component but adds its own functionality, and does not need the seek functionality.
       // This is actually a hack, the proper solution would be for both seek bar and volume sliders to extend
       // a common base slider component and implement their functionality there.
+
       return;
```

Comment:
* A `SeekBar` should not configure a `VolumeSlider` - they are totally unrelated
 * Why is the added code above the describing comment of the conditional flag `configureSeek`?
 * Why is there a blank line added?

Reply:
https://github.com/bitmovin/bitmovin-player-ui/pull/282#discussion_r354857696

Reply:
I have seen this comment - but that doesn't make the issue go away ;) I'm sure this can be written in a better way.

Reply:
Only way I see is separating the seek slider from the volume slider, not extending one from another like we do now in this case

Reply:
What about a `protected configureAria(...) {. .. }` method that can be overwritten in the volume slider?